kombu_fernet/serializers/__init__.py

```diff
@@ -2,15 +2,24 @@
 from __future__ import unicode_literals, absolute_import
 
 import os
-from cryptography.fernet import Fernet, InvalidToken
+from cryptography.fernet import Fernet, MultiFernet
 
 fernet = Fernet(os.environ['KOMBU_FERNET_KEY'])
+fallback_fernet = None
+try:
+    fallback_fernet = Fernet(os.environ['OLD_KOMBU_FERNET_KEY'])
+except KeyError:
+    pass
+else:
+    fernet = MultiFernet([fernet, fallback_fernet])
```

Comment:
Yeah, I understand that, but that'll be needed when we bump the version here and prepare to add to our application requirements.txt